extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,139 @@
+}
+
+.i-amp-story-quiz-option {
+    display: flex !important;
+    justify-items: start !important;
+    align-items: center !important;
+    border-radius: 32px !important;
+    padding: 8px 16px 8px 8px !important;
+    background-color: inherit !important;
+    margin-bottom: 8px !important;
+    color: #5F6368 !important;
+    border: solid 1px #DADCE0 !important;
+    /* TODO: Why won't this notation work? */
+    /* font: 16px/20px !important; */
+    font-size: 16px !important;
+    line-height: 20px !important;
+}
+
+/* Truncate text and add an ellipsis after 2 lines. */
+.i-amp-story-quiz-option-text {
+    max-height: 40px !important;
+    overflow : hidden !important;
+    text-overflow: ellipsis !important;
+    display: -webkit-box !important;
+    -webkit-line-clamp: 2 !important;
+    -webkit-box-orient: vertical !important;
+    visibility: visible !important;
+}
+
+.i-amp-story-quiz-post-selection .i-amp-story-quiz-option-selected {
+    color: white !important;
+    border: solid 1px transparent !important;
+}
+
+.i-amp-story-quiz-answer-choice {
+    display: flex !important;
+    justify-content: center !important;
+    align-items: center !important;
+    height: 24px !important;
+    width: 24px !important;
+    min-width: 24px !important;
+    border-radius: 50% !important;
+    border: solid 2px !important;
+    padding: 5px !important;
+    margin-right: 16px !important;
+    color: var(--accent-color) !important;
+    font-size: 16px !important;
+    line-height: 16px !important;
+    font-weight: bold !important;
+}
+
+[dir=rtl ] .i-amp-story-quiz-answer-choice {
+    margin-left: 16px !important;
+    margin-right: 0px !important;
+}
+
+.i-amp-story-quiz-post-selection :not([correct]) > .i-amp-story-quiz-answer-choice {
+    color: var(--incorrect-color) !important;
+}
+
+.i-amp-story-quiz-post-selection [correct] > .i-amp-story-quiz-answer-choice {
+    color: var(--correct-color) !important;
+}
+
+.i-amp-story-quiz-post-selection .i-amp-story-quiz-option-selected:not([correct]) {
+    background-color: var(--incorrect-color) !important;
+}
+
+.i-amp-story-quiz-post-selection .i-amp-story-quiz-option-selected[correct] {
+    background-color: var(--correct-color) !important;
+}
+
+.i-amp-story-quiz-post-selection .i-amp-story-quiz-option-selected > .i-amp-story-quiz-answer-choice {
+    border: solid 2px transparent !important;
+}
+
+/** TODO(jackbsteinberg): is font-size: 0; a11y friendly? */
+
+.i-amp-story-quiz-post-selection [correct] > .i-amp-story-quiz-answer-choice {
+    background-color: white !important;
+    background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="#5BBA74"><path fill="none" d="M0 0h24v24H0z"/><path d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"/></svg>') !important;
+    background-repeat: no-repeat !important;
+    background-position: center !important;
+    font-size: 0 !important;
```

Comment:
What does this do?

Reply:
Oh, got it!
For the record, another more a11y friendly approach would have been to have different containers for the A/B/C/D and the success/error icons. You can then set a `display: block/none;` and the corresponding `aria-hidden`/`aria-label`/`aria-live` attributes.

Reply:
Thanks for the suggestion - I added this to the list of a11y priorities to be taken care of before launch